### DISCUSSION TITLE
Missing definitions in [@types/ckeditor__ckeditor5-list] for DocumentList & DocumentListProperties

### DISCUSSION THREAD

**Discussion by tony1377** [DefinitelyTyped/DefinitelyTyped#63357] [Issues with a @types package]
The @ckeditor/ckeditor5-list package exports `DocumentList` & `DocumentListProperties` but the equivalent @types definition does not currently include those

![image](https://user-images.githubusercontent.com/4586382/203381114-a320fe14-bef2-459f-b5e2-4584530dc90b.png)

[a comment by an automated account is not shown]

**Comment by fedemp** [comment]
Kind of expected cause current definitions match an older version than current. Given that @ckeditor is working on a migration to TS, I have chosen to focus more on my `${work} now.

**Comment by martin-badin** [comment]
PR is ready https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63556
